### DISCUSSION TITLE
Sonos Port

### DISCUSSION THREAD

**Discussion by thephantom** [punxaphil/custom-sonos-card#536] [Q&A]
Hi,

Thanx for the great solution. I have also an Sonos port (https://www.sonos.com/nl-nl/shop/port) for playing my LP records. 

Is it possible that I can also control this so that I can get music from the Line in From the port on a certain speaker? Unfortunately, grouping does not work. The port is visible in Home Assistant.

**Comment by punxaphil** [comment]
If you can control it using the built in media player, then it should work with the sonos card.
Go to Settings -> Devices -> Select your device
On this page you can see what's possible. 
Another way is to go to the Media page in the sidebar.

**Comment by thephantom** [comment]
Is is this part from the sonos app
________________________________
Van: Punxa Phil ***@***.***>
Verzonden: Tuesday, November 12, 2024 8:07:20 AM
Aan: punxaphil/custom-sonos-card ***@***.***>
CC: thephantom ***@***.***>; Author ***@***.***>
Onderwerp: Re: [punxaphil/custom-sonos-card] Sonos Port (Discussion #536)


If you can control it using the built in media player, then it should work with the sonos card.
Go to Settings -> Devices -> Select your device
On this page you can see what's possible.
Another way is to go to the Media page in the sidebar.

**Reply by punxaphil** [reply]
??
